### PR TITLE
[FEATURE] fsa styles applied. styles are checked in and a script has been left to update these if needed

### PR DESCRIPTION
**Pull request by danWatkinson**
…eed to get into a place where we just check in the compiled styeles rather than compiling them ourselves

### CODE REVIEW

File: incident-application-form/package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
+    "updateFSAStyles": "npm install fsastyles && cd ./node_modules/fsastyles && npm install && npm run build && cp -R ./dist/* ../../public",
```
Comment (danWatkinson) [inline incident-application-form/package.json:6]: hack that lets us claim to be able to update fsa styles easily if needed..

can probably be done via webpack by people who know how, but would require us to have fsastyles as an actual dependency
having fsastyles as an actual dependency adds 1 minute to a 2 1/2 minute CI process.

File: incident-application-form/public/app.css [hunk 2]
```diff
@@ -0,0 +1 @@
...ld_content_promo__item .promo__intro,.promo .field__field_intro,.promo .promo__intro{padding-left:2em}}@media (min-width:30em){.field__field_content_promo__item .field__field_intro,.field__field_content_promo__item .promo__intro,.promo .field__field_intro,.promo .promo__intro{padding-bottom:1.25rem}}@media (min-width:50em){.field__field_content_promo__item .field__field_intro,.field__field_content_promo__item .promo__intro,.promo .field__field_intro,.promo .promo__intro{padding-bottom:1.5rem}}@media (min-width:60em){.field__field_content_promo__item .field__field_intro,.field__field_content_promo__item .promo__intro,.promo .field__field_intro,.promo .promo__intro{padding-bottom:1.75rem}}@media (min-width:80em){.field__field_content_promo__item .field__field_intro,.field__field_content_promo__item .promo__intro,.promo .field__field_intro,.promo .promo__intro{padding-bottom:2rem}}.field__field_content_promo__item .field__field_intro .field__field_intro,.field__field_content_promo__item .promo__intro .field__field_intro,.promo .field__field_intro .field__field_intro,.promo .promo__intro .field__field_intro{padding:0}.field__field_large_promo__item .field__field_intro,.field__field_lar
```
Comment (danWatkinson) [inline incident-application-form/public/app.css:1]: not ideal but these are all the files that fsastyles generates it when you 'npm run build' on it and we want to serve them up statically so.. here they are.
script ^ will re-do these if fsastyles change..

File: incident-application-form/views/layout.njk [hunk 3]
```diff
@@ -5,6 +5,7 @@
 
 {% block head %}
     {{ super() }}
+    <link rel="stylesheet" type="text/css" href="/app.css">
```
Comment (danWatkinson) [inline incident-application-form/views/layout.njk:8]: add the fsa styles

File: test/cypress/integration/user_provides_contact_details.spec.js [hunk 4]
```diff
@@ -15,32 +15,32 @@ context(
     it(`Visit ${YOUR_DETAILS}, fill in some valid data, submit.`, () => {
       cy.visit(YOUR_DETAILS);
 
-      cy.get('[data-cy="notifier-type"]').select("Industry");
+      cy.get('[data-cy="notifier-type"]').select("Industry", {force: true});
```
Comment (danWatkinson) [inline test/cypress/integration/user_provides_contact_details.spec.js:18]: cypress goes mad when the fsa styles get put in place.
it runs a load of usability checks before it interacts with anything and it's failing on a bunch of the fsa-styled components now we've brought in their css
{force:true} on every interaction tells cypress to quit it's whining about usability and just select the thing/fill in the field/click the button/whatever.

i had a brief google for how to make cypress do that on everything by default to make it a one-liner but couldn't find anything and i've already done it now..

**Review by a6software — APPROVED** [review]

**Pull request merged**